packages/design-system-docs/src/pages/startup/sass-and-css.md

```diff
@@ -3,38 +3,95 @@ title: Using Sass/CSS
+By default, the design system expects a file structure like this:
 
-1. First, make sure your build system is configured so that the `node_modules` directory is in the list of Sass [`includePaths`](https://github.com/sass/node-sass#includepaths).
-2. Add the following to your Sass file:
+```
+└── Your site's build folder/
+    ├── images/
+    ├── fonts/
+    └── index.css
+```
+
+You can manually copy the image and font directories, or you could integrate this step into your build process.
+
+- [Here's an example of copying assets using Gulp](https://github.com/CMSgov/design-system/blob/master/examples/react-app/Gulpfile.js).
+- [Here's an example of copy assets using Webpack](https://github.com/CMSgov/design-system/blob/master/examples/webpack-demo/webpack.config.js).
+
+Alternatively if your project uses Webpack or Create React App, you can [configure the bundler to process the assets]({{root}}/startup/sass-and-css/#setting-asset-paths-to-node_modules).
+
+### Overriding asset paths
+
+If you are using Sass/SCSS, you can change the default asset paths by overriding the `$font-path` and `$image-path` Sass variables.
+
+For example, if your project build directory file structure looked like this:
+
+```
+└── Your site's build folder/
+    ├── images/
+    ├── fonts/
+    └── css/
+      └── index.css/
+```
+
+The asset paths would have to be redefined like so:
 
 ```css
-@import '{{npm}}/dist/scss/index';
+$font-path: '../fonts';
+$image-path: '../image';
 ```
 
-[Learn how to override and theme Sass variables]({{root}}/startup/theming/).
+Without overriding these variables, it's possible that your fonts and images will not render correctly in your project.
+
+#### Setting asset paths to `node_modules`
+
+If your project uses Webpack or Create React App, you can set the asset paths directly to the fonts and image folders in your `node_modules`.
+
+```css
+$font-path: '~{{npm}}/dist/fonts';
+$image-path: '~{{npm}}/dist/images';
+```
+
+This approach is recommended, as you no longer have to copy assets into your project. It does however require some additional configuration.
+
+For Create React App projects, simply add the following to your `.env`. This will allow Create React App's Webpack configuration to know to process the fonts and images in `node_modules`.
+
+```
+SASS_PATH="node_modules"
+```
 
-[View an example](https://github.com/CMSgov/design-system/blob/master/examples/react-app/src/styles/index.scss)
+If you have your own custom Webpack configuration, you need to use [`file-loader`](https://webpack.js.org/loaders/file-loader/) and [`resolve-url-loader`](https://github.com/bholloway/resolve-url-loader) to copy assets into your build folder and inject the correct URLS to your build files.
 
-<h4 class="ds-h4 ds-u-font-size--base">Applying styles to your page</h4>
+## Applying styles to your page
```

Comment:
the `utility class` link below should be updated to `{{root}}/utilities/overview/`